packages/block-library/src/buttons/transforms.js

```diff
@@ -26,6 +26,42 @@ const transforms = {
 					)
 				),
 		},
+		{
+			type: 'block',
+			isMultiBlock: true,
+			blocks: [ 'core/paragraph' ],
+			transform: ( buttons ) =>
+				// Creates the buttons block
+				createBlock(
+					name,
+					{},
+					// Loop the selected buttons
+					buttons.map( ( attributes ) => {
+						// Remove any HTML tags
+						const div = document.createElement( 'div' );
+						div.innerHTML = attributes.content;
+						const text = div.innerText || '';
+						// Get first url
+						const link = div.querySelector( 'a' );
+						const url = link ? link.getAttribute( 'href' ) : null;
+						// Create singular button in the buttons block
+						return createBlock( 'core/button', {
+							text,
+							url,
+						} );
+					} )
+				),
+			isMatch: ( paragraphs ) => {
+				const buttons = paragraphs.filter( ( attributes ) => {
```

Comment:
You can use [Array.prototype.every()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) here.